doc/news/changes/minor/20241120tjhei

```diff
@@ -0,0 +1,7 @@
+New: FiniteElement::get_local_dof_sparsity_pattern() can be used to
+provide the coupling between DoFs within a cell and is used in
+make_sparsity_pattern() to generate matrices with fewer nonzero
+entries depending on the element. FE_Q_iso_Q1 now provides this
```

Comment:
An extension to `FESystem` would be nice :)

Reply:
On our list!